### PR TITLE
Task 2:

### PR DESCRIPTION
**Pull request by Lindar90**
- fix image placeholder
- change store name
- bump serverless dependencies
- automated deployment (S3 bucket, cloudfront, cache invalidation)

### CODE REVIEW

**Comment by mabioIa** [comment]
Where is the location of your serverless-single-page-app-plugin?

File: serverless.yml [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
-  # setup profile for AWS CLI.
-  # profile: node-aws
+  region: eu-central-1
 
 plugins:
   - serverless-finch
-  # 'serverless-single-page-app-plugin' is a custom plugin that located .serverless_plugins folder.
-  # Existing plugin (https://www.npmjs.com/package/serverless-single-page-app-plugin) doesn't have invalidate cache feature that often used during CI/CD events.
-  # How to build your own plugins: https://www.serverless.com/framework/docs/providers/aws/guide/plugins#service-local-plugin
   - serverless-single-page-app-plugin
```
Comment (MedvedevaM) [inline serverless.yml:30]: - serverless-s3-cleaner

You can also add this plugin with some more lines (I'll add them above) and then add a script to package.json for removing all the artifacts (something like that "serverless:remove": "sls remove")

Comment (Lindar90) [reply serverless.yml:30]: Thank you for you advice! I will give it a try

Comment (ts4ed) [reply serverless.yml:30]: Hello, where are the links to s3 and cloudFront?

Comment (Lindar90) [reply serverless.yml:30]: @ts4ed in Readme file as it was stated in task description

Comment (ts4ed) [reply serverless.yml:30]: OK, I found

CloudFront URL: 'https://dccjg46zrbytv.cloudfront.net/'
S3-website: 'https://shop-vue-vuex-cloudfront-dzianis.s3.eu-central-1.amazonaws.com/index.html'

But link to S3 does not throw a 403 error

Comment (Lindar90) [reply serverless.yml:30]: @ts4ed 
image

Could I ask you to check Network panel? The status code is 403 for the given URL

Comment (ts4ed) [reply serverless.yml:30]: Ah, okay. It's it's my fail
I was expecting this )
image
Good job. Good luck

Comment (Lindar90) [reply serverless.yml:30]: @ts4ed no problem. I suppose that's a question of configuration of the bucket. Probably there is some checkbox that allows to change the access deny output. Thank you for your attention!

**Pull request merged**